node/src/server-modules/GlideJson.ts

```diff
@@ -231,10 +231,81 @@ export class GlideJson {
     ): Promise<ReturnTypeJson<boolean>> {
         const args = ["JSON.TOGGLE", key];
 
-        if (options !== undefined) {
+        if (options) {
             args.push(options.path);
         }
 
         return _executeCommand<ReturnTypeJson<boolean>>(client, args);
     }
+
+    /**
+     * Deletes the JSON value at the specified `path` within the JSON document stored at `key`.
+     *
+     * @param client - The client to execute the command.
+     * @param key - The key of the JSON document.
+     * @param options - (Optional) Additional parameters:
+     * - (Optional) If `null`, deletes the entire JSON document at `key`.
```

Comment:
```suggestion
     * - path (Optional): If `null`, deletes the entire JSON document at `key`.
```

Reply:
I realize in previous commands we didn't name the optional parameters. Can we clean that up too?